md/ubuntu-openjdk-tomcat-postgresql.md

```diff
@@ -136,45 +138,72 @@ Warning: make sure you stop Tomcat before performing following operations: `sudo
 * Edit `bitronix-resources.properties` file, comment (using `#`) h2 section, uncomment PostgreSQL example
 and update the username, password and DB name ("bonita" in the URL property) to match your configuration (e.g.
 `bonita_db_user`, `bonita_db_password` and `bonita_db`): `sudo nano /opt/bonita/btm/conf/bitronix-resources.properties`
 * Also in `bitronix-resources.properties` update the section for `resource.ds2` (BDM data source) and set the value for the BDM data base (e.g. `bonita_db_user`, `bonita_db_password` and `bonita_bdm`)
 * Save and quit: `CTRL+X, Y, ENTER`
-* Copy the `bonita.xml` file (Bonita web app context configuration): `sudo cp ~/BonitaBPMSubscription-6.4.0-deploy/Tomcat-7.0.55/conf/Catalina/localhost/bonita.xml /etc/tomcat7/Catalina/localhost/`
+* Copy the `bonita.xml` file (Bonita web app context configuration): `sudo cp ~/BonitaBPMSubscription-x.y.z-deploy/Tomcat-7.0.67/conf/Catalina/localhost/bonita.xml /etc/tomcat7/Catalina/localhost/`
 * Edit the `bonita.xml` file by commenting the h2 datasource configuration (using ),
 uncomment PostgreSQL example and update username, password and DB name (bonita in the URL property) to match your
 configuration (e.g. `bonita_db_user`, `bonita_db_password` and `bonita_db`): `sudo nano /etc/tomcat7/Catalina/localhost/bonita.xml`
 * Also in `bonita.xml` file update data base configuration for BDM to match your configuration (e.g. `bonita_db_user`, `bonita_db_password` and `bonita_bdm`)
 * Save and quit: `CTRL+X, Y, ENTER`
-* Copy and overwrite `logging.properties` file: `sudo cp ~/BonitaBPMSubscription-6.4.0-deploy/Tomcat-7.0.55/conf/logging.properties /etc/tomcat7/logging.properties`
-* Copy and overwrite `context.xml` file: `sudo cp ~/BonitaBPMSubscription-6.4.0-deploy/Tomcat-7.0.55/conf/context.xml /etc/tomcat7/context.xml`
-* Copy and overwrite `server.xml` file: `sudo cp ~/BonitaBPMSubscription-6.4.0-deploy/Tomcat-7.0.55/conf/server.xml /etc/tomcat7/server.xml`
+* Copy and overwrite `logging.properties` file: `sudo cp ~/BonitaBPMSubscription-x.y.z-deploy/Tomcat-7.0.67/conf/logging.properties /etc/tomcat7/logging.properties`
+* Copy and overwrite `context.xml` file: `sudo cp ~/BonitaBPMSubscription-x.y.z-deploy/Tomcat-7.0.67/conf/context.xml /etc/tomcat7/context.xml`
+* Copy and overwrite `server.xml` file: `sudo cp ~/BonitaBPMSubscription-x.y.z-deploy/Tomcat-7.0.67/conf/server.xml /etc/tomcat7/server.xml`
 * Edit `server.xml` (`sudo nano /etc/tomcat7/server.xml`) and comment out h2 listener line
 * Fix ownership on the copied files: `sudo chown -R root:tomcat7 /etc/tomcat7`
+* Copy the folder `platform_setup`: `sudo cp -R ~/BonitaBPMSubscription-x.y.z-deploy/platform_setup /opt/bonita/` and rename it: `sudo mv /opt/bonita/platform_setup /opt/bonita/setup`
+* Fix ownership on the copied files: `sudo chown -R tomcat7:tomcat7 /opt/bonita/setup`
+
+### Fix service tomcat7 issue
```

Comment:
I can't tell actually. I spent a fair amount of time trying to figure a better way without success. The issue with Tomcat 7 in service mode is that it doesn't seem to execute the setenv.sh file, so the environment variables are not correctly set. 
Maybe it's related to my Linux distrib, or the Tomcat version, but that's the only workaround I found.